src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -803,6 +806,8 @@
     text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-keda
   - name: Disable KEDA workload autoscaler for an existing kubernetes cluster.
     text: az aks update -g MyResourceGroup -n MyManagedCluster --disable-keda
+  - name: Update the SSH key on a AKS Cluster.
+    text: az aks update -g MyResourceGroup -n MyManagedCluster --ssh-key-value "<new SSH key value or SSH key file>"    
```

Comment:
```suggestion
    text: az aks update -g MyResourceGroup -n MyManagedCluster --ssh-key-value /path/to/publickey
```

Reply:
This code suggests attempting to resolve CI issues and maintaining consistency with [aks-preview](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/_help.py#L533)